### PR TITLE
Config changes from develop

### PR DESCRIPTION
**Pull request by sowmya695**

### CODE REVIEW

File: registration-processor-default.properties [hunk 1]
```diff
@@ -450,7 +448,7 @@ mosip.auth.adapter.impl.basepackage=io.mosip.kernel.auth.defaultadapter
 
 # BioSDK 
 #mosip.biosdk.default.host=${mosip.regproc.biosdk.url}
-mosip.biosdk.default.service.url=${mosip.regproc.biosdk.url}/biosdk-service
+mosip.biosdk.default.service.url=${mosip.mock.biosdk.url}//biosdk-service
```
Comment (MonobikashDas) [inline registration-processor-default.properties]: remove one extra slash /

File: application-default.properties [hunk 2]
```diff
@@ -40,7 +40,6 @@ mosip.idrepo.credrequest.generator.url=http://credentialrequest.idrepo
 mosip.idrepo.credential.service.url=http://credential.idrepo
 mosip.datashare.url=http://datashare.datashare
 mosip.mock.biosdk.url=http://biosdk-service.biosdk
-mosip.regproc.biosdk.url=http://biosdk-service.biosdk
```
Comment (MonobikashDas) [inline application-default.properties:43]: why did u remove it?

File: registration-processor-default.properties [hunk 3]
```diff
@@ -479,7 +477,6 @@ mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.format.url.mock-1.1
 ## Printing
 mosip.registration.processor.credentialtype=euin
 mosip.registration.processor.encrypt=false
-mosip.registration.processor.issuer=mpartner-default-print
```
Comment (MonobikashDas) [inline registration-processor-default.properties:482]: i see we use this config in code. why it was removed?

**Review by MonobikashDas — CHANGES_REQUESTED** [review]

File: application-default.properties (hunk 2 above)
Comment (sowmya695) [reply application-default.properties:43]: It is not used in regproc as common mock sdk url updated in repgroc

File: registration-processor-default.properties (hunk 1 above)
Comment (sowmya695) [reply registration-processor-default.properties]: fixed

File: registration-processor-default.properties (hunk 3 above)
Comment (sowmya695) [reply registration-processor-default.properties:482]: Added updated config at the end which is changes done by Dhanendra

**Review by MonobikashDas — APPROVED** [review]

**Pull request merged**